src/Disks/VolumeJBOD.h

```diff
@@ -38,11 +55,18 @@ class VolumeJBOD : public IVolume
     /// Returns valid reservation or nullptr if there is no space left on any disk.
     ReservationPtr reserve(UInt64 bytes) override;
 
+    bool areMergesAllowed() const override;
+
+    void setAllowMergesUserOverride(bool allow) override;
+
+    /// True if parts on this volume participate in merges according to configuration.
+    bool are_merges_allowed = true;
```

Comment:
shouldn't this and below be hidden/private?

Reply:
> We shall not merge that for a while, I have few failing tests on my local machine.

My failing tests were caused by OOMs, everything is fine now.

Reply:
Yep

Reply:
I would like to make it in separate PR because that involves some changes in `VolumeRAID1`/`createVolumeFromReservation` and some discussion of that. Ideally, as I can see, we shall separate `VolumeSingleDisk` and all volumes created from reservation to separate set of classes (from `VolumeJBOD` and `VolumeRAID1`).

Reply:
>involves some changes in VolumeRAID1/createVolumeFromReservation and some discussion of that. Ideally

I don't see any usage of those in the VolumeRAID1 only in `createVolumeFromReservation`
Can `createVolumeFromReservation` use `VolumeJBOD::areMergesAllowed()` instead? AFAICS `are_merges_allowed` is not used directly anywhere.

Reply:
@azat No, it can not, because it is logically different things, `areMergesAllowed()` is the effective value and member `are_merges_allowed` is setting from configuration.

Reply:
@excitoon Ok. But is this the intention that it will not apply `SYSTEM START/STOP MERGES` for writing temporary parts?

AFAICS this does not makes any difference, since temporary parts should not be merged anyway, maybe `createVolumeFromReservation` should always disable merges? since it is kind of tricky anyway

Correct me if I'm wrong.